testproject/jijtest/build.gradle

```diff
@@ -2,10 +2,27 @@ plugins {
     id 'net.neoforged.moddev'
 }
 
+sourceSets {
+    service
+}
+
+def serviceJar = tasks.register("serviceJar", Jar) {
+    from(sourceSets.service.output)
+    archiveClassifier = "service"
+    manifest {
+        attributes(
+            'FMLModType': 'LIBRARY',
+            'Automatic-Module-Name': 'jij.service'
+        )
+    }
+}
+
 dependencies {
+    implementation files(sourceSets.service.output)
```

Comment:
I wonder if this leaks to `runtimeElements`.

Reply:
No, `FileCollection` dependencies do not get published